src/scripts/main.js

```diff
@@ -1,7 +1,82 @@
-'use strict';
-
 // Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+import Game from './modules/Game.class';
+
+const game = new Game();
+
+// listening the classes from html files
+
+const scoreDisplay = document.querySelector('.game-score');
+const startButton = document.querySelector('.start');
+const fieldCell = document.querySelector('.field-cell');
```

Comment:
The `fieldCell` variable should be initialized using `document.querySelectorAll` instead of `document.querySelector` to select all elements with the class 'field-cell'. This is necessary because there are multiple cells in the game grid.